Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Base image
+FROM ubuntu:20.04
+
+# Set working directory
+WORKDIR /app
+
+# Update and install required packages
+RUN apt-get update && \
+    apt-get install -y git-lfs wget && \
+    rm -rf /var/lib/apt/lists/*
+
+# Download and install Miniconda
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh && \
+    bash Miniconda3-latest-Linux-x86_64.sh -b -p /opt/conda && \
+    rm Miniconda3-latest-Linux-x86_64.sh
+
+# Set conda to automatically activate base environment on login
+RUN echo ". /opt/conda/etc/profile.d/conda.sh" >> ~/.bashrc && \
+    echo "conda activate base" >> ~/.bashrc
+
+# Create OpenChatKit environment
+COPY environment.yml .
+RUN conda env create -f environment.yml
```

Comment:
Perhaps we should use `mamba` here instead. This can take a very long time to build the image.

Reply:
This takes a while (forever) for me. Meanwhile it doesn't recognize conda yet, have to use `/opt/conda/bin/conda` instead of `conda`.